input/elasticapm/internal/modeldecoder/v2/span_test.go

```diff
@@ -174,9 +174,16 @@ func TestDecodeMapToSpanModel(t *testing.T) {
 		assert.Equal(t, "failure", out.Event.Outcome)
 		// derive from other fields - unknown
 		input.Outcome.Reset()
+		input.OTel.Reset()
 		input.Context.HTTP.StatusCode.Reset()
 		mapToSpanModel(&input, &out)
 		assert.Equal(t, "unknown", out.Event.Outcome)
+		// outcome is success when not assigned and it's otel
+		input.Outcome.Reset()
+		input.OTel.SpanKind.Set(spanKindInternal)
```

Comment:
The change was only made to OTel input for now, if otel attributes are set, it will use the same logic for mapping the Unset status to success outcome